packages/client/src/components/app/forms/MultiFieldSelect.svelte

```diff
@@ -61,17 +63,31 @@
   bind:fieldSchema
 >
   {#if fieldState}
-    <CoreMultiselect
-      value={fieldState.value || []}
-      error={fieldState.error}
-      getOptionLabel={flatOptions ? x => x : x => x.label}
-      getOptionValue={flatOptions ? x => x : x => x.value}
-      id={fieldState.fieldId}
-      disabled={fieldState.disabled}
-      on:change={handleChange}
-      {placeholder}
-      {options}
-      {autocomplete}
-    />
+    {#if !optionsType || optionsType === "select"}
```

Comment:
Well done for remembering to make select the fallback to account for all existing apps.